### PR TITLE
Rename storage.conf(5) to containers-storage.conf(5)

### PR DESCRIPTION
**Pull request by rhatdan**
We have a conflict on man pages storage.conf man page is owned
by the inn package, so we need to pick a less generic name.

Signed-off-by: Daniel J Walsh <dwalsh@redhat.com>

### CODE REVIEW

**Comment by rhatdan** [comment]
retest

**Comment by TomSweeneyRedHat** [comment]
The change LGTM, but something wonky is going on in dockerCmd_utils_test.go:270.

**Comment by rhatdan** [comment]
Tests are now passing.

@nalind PTAL

File: storage.conf [hunk 1]
```diff
@@ -1,8 +1,9 @@
+# See man 5 containers-storage.conf for mor information
```
Comment (nalind) [inline storage.conf]: Typo: `mor` should be `more`.

**Comment by nalind** [comment]
Did the bits that change the name of the file fall out of this, or is the description no longer correct?

File: storage.conf (hunk 1 above)
Comment (rhatdan) [reply storage.conf]: Fixed

**Comment by rhatdan** [comment]
@nalind PTAL
The rename of the man page still exists.

**Comment by nalind** [comment]
LGTM, merging.

**Pull request merged**